Prox/Prox/PlaceCarousel/PlaceCarouselCollectionViewCell.swift

```diff
@@ -0,0 +1,97 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+import QuartzCore
+
+class PlaceCarouselCollectionViewCell: UICollectionViewCell {
+
+    lazy var roundedBackgroundView: UIView = {
+        let view = UIView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.backgroundColor = .white
+        view.layer.cornerRadius = 5
+        view.clipsToBounds = true
+        return view
+    }()
+
+    lazy var placeImage: UIImageView = {
+        let imageView = UIImageView()
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.contentMode = .scaleAspectFill
+
+        let opacityView = UIView()
+        opacityView.backgroundColor = UIColor(colorLiteralRed: 0.0, green: 0.0, blue: 0.0, alpha: 0.1)
```

Comment:
What we are doing here is adding a slightly opaque dark masque to the top of the image, making it dark enough to see the white text on top, as opposed to making the image itself less opaque, which is what changing the alpha value of the `UIImageView` would do